modules/tinymce/src/core/test/ts/browser/keyboard/EnterKeyTest.ts

```diff
@@ -718,31 +727,43 @@ describe('browser.tinymce.core.keyboard.EnterKey', () => {
     const editor = hook.editor();
     editor.getBody().innerHTML = '<ul><li><p><br /></p></li><li><p>b</p></li><li>c</li></ul>';
     LegacyUnit.setSelection(editor, 'p', 0);
-    pressEnter(editor);
+    pressEnter(editor, true);
     assert.equal(editor.getContent(), '<p>\u00a0</p><ul><li><p>b</p></li><li>c</li></ul>');
   });
 
   it('Enter inside middle li with block inside', () => {
     const editor = hook.editor();
     editor.getBody().innerHTML = '<ul><li>a</li><li><p><br /></p></li><li>c</li></ul>';
     LegacyUnit.setSelection(editor, 'p', 0);
-    pressEnter(editor);
+    pressEnter(editor, true);
     assert.equal(editor.getContent(), '<ul><li>a</li></ul><p>\u00a0</p><ul><li>c</li></ul>');
   });
 
   it('Enter inside last li with block inside', () => {
     const editor = hook.editor();
     editor.getBody().innerHTML = '<ul><li>a</li><li>b</li><li><p><br /></p></li></ul>';
     LegacyUnit.setSelection(editor, 'p', 0);
-    pressEnter(editor);
+    pressEnter(editor, true);
     assert.equal(editor.getContent(), '<ul><li>a</li><li>b</li></ul><p>\u00a0</p>');
   });
 
   it('Enter inside summary element', () => {
     const editor = hook.editor();
     editor.getBody().innerHTML = '<details><summary>ab</summary></details>';
     LegacyUnit.setSelection(editor, 'summary', 1);
-    pressEnter(editor);
+    pressEnter(editor, false);
     assert.equal(editor.getContent(), '<details><summary>a<br />b</summary></details>');
   });
+
+  it('Enter on expanded range', () => {
+    const editor = hook.editor();
+    const apis = TinyApis(editor);
+
+    editor.setContent('<p>abc</p>');
+    apis.setSelection([ 0, 0 ], 1, [ 0, 0 ], 2);
```

Comment:
```suggestion
    TinySelections.setSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
```
This is how the newer BDD APIs are used.

Reply:
Nice, didn't know about these ones good to know for future PRs.

Reply:
For what it's worth or if it's helpful, https://github.com/tinymce/tinymce/blob/develop/modules/mcagar/docs/bdd.md has the details on the new BDD APIs :shrug: